docs/docs/usage/scaladoc/settings.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Scaladoc settings
+---
+
+# {{page.title}}
+
+You can find brief information included in this chapter while calling scaladoc with `-help` flag. However it can be a little tricky if one uses sbt `doc` task.
+
+## Parity with scaladoc for Scala 2
+
+Scaladoc has been rewritten from scratch and some of the features turned out to be useless in the new context.
+If you want to know what is current state of compatibility with scaladoc old flags, you can visit this issue for tracking [progress](https://github.com/lampepfl/dotty/issues/11907).
```

Comment:
See my comment on that issue, I think we should move it to the migration guide: https://github.com/scala/docs.scala-lang/tree/main/_overviews/scala3-migration

Reply:
Do you think we should move it (and remove it here) or just update migration guide with similar notice?

Reply:
I can do it any way, of course, just let me know

Reply:
I think eventually we want to move the content of dotty.epfl.ch into docs.scala-lang.org. But this page (settings.md) should still contain this sentence with a link to the matrix. However, the matrix itself should be moved from a github issue to a page in the migration guide. Let me know if I’m not clear :)

Reply:
I’m adding a note here to not forget to update this link after https://github.com/scala/docs.scala-lang/pull/2012 is merged.